playerSDK/src/main/java/com/kaltura/playersdk/PlayerViewController.java

```diff
@@ -702,254 +586,254 @@ public String toString() {
-                                                                } else {
-                                                                    wvLp.height = getLayoutParams().height;
-                                                                }
-
-                                                                updateViewLayout(mWebView, wvLp);
-                                                            }
-
-                                                        }
-
-                                                    });
-                                                    mVideoInterface.play();
-                                                } else {
-                                                    Log.w(TAG, "DoubleClick is not supported by this player");
-                                                }
-                                            }else if(params.get(0).equals("goLive")){
-                                                Log.d(TAG, "SEEK: GOTOLIVE");
-                                                //temporary fix - waiting for an HLS api
-                                                if (params.get(1).equals("true") && mVideoInterface instanceof LiveStreamInterface){
-                                                    ((LiveStreamInterface)mVideoInterface).switchToLive();
-                                                }
-                                            } else if (params.get(0).equals("nativeAction")) {
-                                                doNativeAction(params.get(1));
-                                            }
-                                        }
-                                    } else if (action.equals("notifyKPlayerEvent")) {
-                                        return notifyKPlayerEvent(value, mKplayerEventsMap, false);
-                                    } else if (action.equals("notifyKPlayerEvaluated")) {
-                                        return notifyKPlayerEvent(value, mKplayerEvaluatedMap, true);
-                                    }
-                                }
-
-
-                            } catch (Exception e) {
-                                Log.w(TAG, "action failed: " + action);
-                            }
-                        }
-
-                    }
-                } else {
-                    if (mVideoInterface.canPause()) {
-                        mVideoInterface.pause();
-                        mVideoInterface.setStartingPoint((int) (mCurSec * 1000));
-                    }
-                    Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse( url ));
-                    mActivity.startActivity(browserIntent);
-                    return true;
-                }
-
-            }
-
-            return false;
-        }
+//        @Override
+//        public void onPageFinished(WebView view, String url) {
+//            super.onPageFinished(view, url);
```

Comment:
@itaykinnrot  I don't want to remove all the marked code, because I use it for understand what was the intention of the previous code, when all the operation will be replaced I will remove it